README.md

```diff
@@ -1,53 +1,85 @@
 # Assorted small utilities for MC simulations with Cython.
 
-----------
+[![pip(Linux&MacOS)](https://github.com/ev-br/mc_lib/actions/workflows/pip_ubuntu_macos.yml/badge.svg)](https://github.com/ev-br/mc_lib/actions/workflows/pip_ubuntu_macos.yml/)
+[![meson(Linux & MacOS)](https://github.com/ev-br/mc_lib/actions/workflows/dev_ubuntu_macos.yml/badge.svg)](https://github.com/ev-br/mc_lib/actions/workflows/dev_ubuntu_macos.yml/)
+[![license](http://img.shields.io/badge/license-BSD-blue.svg?style=flat)](https://github.com/ev-br/mc_lib/blob/master/LICENSE)
+
+
+## Installation
+
+There are several ways to install the library. They are all equivalent, and
+only differ in a level of control over the process.
+
 
-### How to install (meson + PEP517)
+### A one-liner
+
+It is easiest to install directly from github:
 
 ```
-    git remote add mc_lib_git https://github.com/ev-br/mc_lib
-    git fetch mc_lib_git
-    git checkout mc_lib_git/master
+$ pip install git+https://github.com/ev-br/mc_lib.git@master
+
 ```
+
+Under the hood we use [`meson`](https://mesonbuild.com/) and PEP517 as a build
+system, but this is well hidden and need not be a concern. 
+Now test your fresh install:
+
 ```
-    pip install .
+$ pytest --pyargs mc_lib -v
 ```
 
-----------
-### How to build (meson)
-Get the latest meson
-```pip install git+https://github.com/mesonbuild/meson.git@master```
+
+### Clone and install
+
+Alternatively, clone the repository from GitHub and run `pip` on the local clone:
+
 ```
-    git remote add mc_lib_git https://github.com/ev-br/mc_lib
-    git fetch mc_lib_git
-    git checkout mc_lib_git/master
+$ git clone git@github.com:ev-br/mc_lib.git
+$ cd mc_lib
+$ pip install .
 ```
+
+
+### Manual build with `meson`
+
+This is needed if you plan to develop the library. First, get the latest `meson`
+snapshot (we hope this is temporary and we'll be able to use a stable version
+once meson 0.60 is released) and other required packages:
+
 ```
-    pip install -r requirements.txt
+$ pip install git+https://github.com/mesonbuild/meson.git@master
```

Comment:
Add here ```git clone https://github.com/ev-br/mc_lib.git ``` before `pip install`?
I think it might be a misunderstanding

Reply:
Yup, good catch! The clone step was missing. fixed now